pkg/get/get_test.go

```diff
@@ -2184,3 +2184,55 @@ func Test_DownloadRekorCli(t *testing.T) {
 	}
 
 }
+
+func Test_DownloadTFSecCli(t *testing.T) {
+	tools := MakeTools()
+	name := "tfsec"
+
+	tool := getTool(name, tools)
+
+	tests := []test{
+		{
+			os:      "darwin",
+			arch:    arch64bit,
+			version: "v0.57.1",
+			url:     `https://github.com/aquasecurity/tfsec/releases/download/v0.57.1/tfsec-darwin-amd64`,
+		},
+		{
+			os:      "darwin",
+			arch:    archARM64,
+			version: "v0.57.1",
+			url:     `https://github.com/aquasecurity/tfsec/releases/download/v0.57.1/tfsec-darwin-arm64`,
+		},
+		{
+			os:      "linux",
+			arch:    arch64bit,
+			version: "v0.57.1",
+			url:     `https://github.com/aquasecurity/tfsec/releases/download/v0.57.1/tfsec-linux-amd64`,
+		},
+		{
+			os:      "linux",
+			arch:    archARM64,
+			version: "v0.57.1",
+			url:     `https://github.com/aquasecurity/tfsec/releases/download/v0.57.1/tfsec-linux-arm64`,
+		},
+		{
+			os:      "ming",
```

Comment:
In a follow-up PR, let's use consts for all these repeated os strings, WDYT?

cc @yankeexe

Reply:
will improve this

Reply:
PR: https://github.com/alexellis/arkade/pull/500